worlds/dkc3/Regions.py

```diff
@@ -44,6 +44,10 @@ def create_regions(world, player: int, active_locations):
         LocationName.lakeside_limbo_bonus_2 : [0x657, 3],
         LocationName.lakeside_limbo_dk      : [0x657, 5],
     }
+    if world.kongsanity[player]:
+        lakeside_limbo_region_locations.update({
```

Comment:
isn't a dealbraker, but more clean would be lakeside_limbo_region_locations[LocationName.lakeside_limbo_kong] = [] for these

Reply:
Unless this is a performance hit doing it this way, I think for file consistency I'd rather leave it this way for now. I may go through this file for cleanup in future, as it is doing a lot of stuff it doesn't need to anymore, but I don't want to break everything right this moment.

Reply:
There likely is a performance hit, as you're creating a dict and then merging two dicts; as opposed to just adding an entry to an existing dict. So there's way more memory shuffling going around, at least one extra container and such.

Reply:
OK, I have resolved this.